### PR TITLE
Adding Attachment schema and tests for BotFile.

### PR DESCRIPTION
**Pull request by RamjotSingh**

### CODE REVIEW

File: CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/CardExtensions.cs [hunk 1]
```diff
@@ -53,5 +53,33 @@ public static Attachment ToAttachment(this O365ConnectorCard card)
                 ContentType = O365ConnectorCard.ContentType
             };
         }
+
+        /// <summary>
+        /// Creates a new attachment from <see cref="FileInfoCard"/>.
+        /// </summary>
+        /// <param name="card"> The instance of <see cref="FileInfoCard"/>.</param>
+        /// <returns> The generated attachment.</returns>
+        public static Attachment ToAttachment(this FileInfoCard card)
```
Comment (punpuny) [inline CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/CardExtensions.cs]: The class description is for O365 and we are putting files card here?

File: CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/TeamsAPI/Models/FileConsentCard.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            Description = description;
+            SizeInBytes = sizeInBytes;
+            AcceptContext = acceptContext;
+            DeclineContext = declineContext;
+        }
+
+        /// <summary>
+        /// Gets or sets file description.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "description")]
```
Comment (punpuny) [inline CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/TeamsAPI/Models/FileConsentCard.cs]: Can we import the Nettonsoft.Json namespace?

Comment (RamjotSingh) [reply CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/TeamsAPI/Models/FileConsentCard.cs]: This file is auto generated. It is generated this way only by autorest.

File: CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/CardExtensions.cs (hunk 1 above)
Comment (RamjotSingh) [reply CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/CardExtensions.cs]: Changing description.

**Pull request merged**